provider/cmd/app/app.go

```diff
@@ -125,22 +127,27 @@ func (o *Options) MarkFlagsRequired(cmd *cobra.Command) {
 	_ = cmd.MarkFlagRequired("supported-machine-classes")
 }
 
-func Command() *cobra.Command {
+// Command creates cobra command
+// It return pointer to pointer for zap logger because flags and logger are initialized in Execute function
+func Command() (*cobra.Command, **originzap.Logger) {
```

Comment:
Double references can be avoided for code simplicity.

Reply:
how?